actor-server/actor-core/src/main/protobuf/dialog.proto

```diff
@@ -239,6 +239,16 @@ message DialogCommands {
 
     message WriteMessageSelfAck {}
 
+    message QuotedMessage {
```

Comment:
Right.
Consider dialogExt as a lightweight interface to `DialogRoot` and `Dialog`. All business-logic contains inside this two actors.